dist/tools/pyterm/pyterm

```diff
@@ -303,6 +303,14 @@ class SerCmd(cmd.Cmd):
         date = time.strftime("%Y-%m-%d %H:%M:%S")
         return ["%s" % (date)]
 
+    def do_PYTERM_sleep(self, line):
+        """Pyterm command: Sleep for n seconds.
+        """
+        if line:
```

Comment:
then all you need is `time.sleep(float(line))`, also why do you need to cast a float, I always though python just figures these things out.

Reply:
When I just do `time.sleep(line)` I get `TypeError: an integer is required (got type str)`

Reply:
For the implementation, I would rather make the time mandatory instead of doing `1` by default.
When I do `sleep` in my shell, I get `sleep: missing operand`.

Reply:
since this is only useful in scripts I think having a default sleep time instead of an error would be more handy.

Reply:
Anyway, I added the Error message.